src/pages/List.js

```diff
@@ -212,7 +212,20 @@ export default function List({ token }) {
         />
         <label htmlFor={doc.id}>{doc.data().item_name}</label>
         <button key={doc.id} onClick={() => deleteItem(doc.id)}>
-          Delete
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            class="h-8 w-8"
+            fill="none"
+            viewBox="0 0 24 24"
+            stroke="white"
+          >
+            <path
+              stroke-linecap="round"
+              stroke-linejoin="round"
+              stroke-width="2"
```

Comment:
Siilar to the chevron-right icon, the SVG properties need to be refactored to the JSX format.

OR  you can just copy and paste the JSX version from Heroicons.com!